readme.md

```diff
@@ -1,13 +1,26 @@
 ## Meetup core
 
 ### Development
-Run the environment:
-```
-docker-compose up -d
-docker-compose run --rm -u "$(id -u):$(id -g)" php composer install
-```
-
-Use Artisan helper:
-```
-docker-compose run --rm -u "$(id -u):$(id -g)" php php artisan
-```
+### Run env for Mac/Linux
```

Comment:
After running `make install` and `make start` on `http://localhost` I see an error
![image](https://user-images.githubusercontent.com/56546832/155536045-b46cc3dd-0fde-429a-ab8f-66c6701d6d90.png)
I think that this problem occurs, because in `.gitignore` we ignore whole `public` directory

What's more, I don't like the readme file.
Sometimes commands are written with dollar, sometimes not. And I believe that we can write "Run environment for Mac/Linux" and not "Run env..." .
Why `Make` commands are uppercase?
Why we have a lot of these commands for setup? Maybe we should think of making one command for setup, where all needed commands will be executed?

Reply:
![Zrzut ekranu (281)](https://user-images.githubusercontent.com/27227621/155896260-51ea5aae-2541-436c-9866-6e5b72db4cbe.png)

Everything works but you have to wait a while until php is fully launched

Reply:
For me not working too.
I have some bugs connected with composer install - and `php` container has restarting.

Reply:
```
Creating meetup_php_run ... done

Warning: require(/application/vendor/autoload.php): Failed to open stream: No such file or directory in /application/artisan on line 8

Fatal error: Uncaught Error: Failed opening required '/application/vendor/autoload.php' (include_path='.:/usr/local/lib/php') in /application/artisan:8
Stack trace:
#0 {main}
  thrown in /application/artisan on line 8
Makefile:2: recipe for target 'install' failed
make: *** [install] Error 255
```

Reply:
I fix problems .Test if it works for you after the changes

Reply:
Install works properly, I have some problems with permissions but its normal if we using docker. I pasted one more error below. On `localhost` should I see something? start page?